### PR TITLE
Update Readme table and loose link to master

### PR DESCRIPTION
**Pull request by vincentpierre**
### Proposed change(s)

Updating the readme table and modifying a link to master in the extensions package.

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,13 +67,13 @@ under active development and may be unstable. A few helpful guidelines:
 | **Version** | **Release Date** | **Source** | **Documentation** | **Download** |
 |:-------:|:------:|:-------------:|:-------:|:------------:|
 | **master (unstable)** | -- | [source](https://github.com/Unity-Technologies/ml-agents/tree/master) | [docs](https://github.com/Unity-Technologies/ml-agents/tree/master/docs/Readme.md) | [download](https://github.com/Unity-Technologies/ml-agents/archive/master.zip) |
```
Comment (vincentpierre) [inline README.md:69]: We could remove master from the table on the release branch.

File: com.unity.ml-agents.extensions/Documentation~/Grid-Sensor.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ These limitations provided the motivation towards the development of the Grid Se
...
 
-Before jumping into the details of the Grid Sensor, an important thing to note is the agent performance and qualitatively different behavior over raycasts. Unity MLAgent's comes with a suite of example environments. One in particular, the [Food Collector](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Learning-Environment-Examples.md#food-collector), has been the focus of the Grid Sensor development.
+Before jumping into the details of the Grid Sensor, an important thing to note is the agent performance and qualitatively different behavior over raycasts. Unity MLAgent's comes with a suite of example environments. One in particular, the [Food Collector](../../docs/Learning-Environment-Examples.md#food-collector), has been the focus of the Grid Sensor development.
```
Comment (vincentpierre) [inline com.unity.ml-agents.extensions/Documentation~/Grid-Sensor.md:39]: Not sure if this is the best approach, but linking to master here will eventually brake.

Comment (chriselion) [reply com.unity.ml-agents.extensions/Documentation~/Grid-Sensor.md:39]: If we ever generate package docs for this, the relative link won't work then either. how about using `release_10_docs`?

File: README.md (hunk 1 above)
Comment (chriselion) [reply README.md:69]: We could, but then we have to remember to re-add it back when merging. I don't think it matters much and isn't worth the headache.

**Review by chriselion — APPROVED** [review]

File: com.unity.ml-agents.extensions/Documentation~/Grid-Sensor.md (hunk 2 above)
Comment (vincentpierre) [reply com.unity.ml-agents.extensions/Documentation~/Grid-Sensor.md:39]: There are some other links in the extensions package that do "../.." should I also link them to release_10_docs? How can I make sure we will update these at each release?

Comment (chriselion) [reply com.unity.ml-agents.extensions/Documentation~/Grid-Sensor.md:39]: > should I also link them to release_10_docs

Sure

> How can I make sure we will update these at each release?

There's a pre-commit hook that looks for these already. You made the changes in https://github.com/Unity-Technologies/ml-agents/pull/4643

**Pull request merged**